packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -99,6 +103,23 @@ A portfolio can be programatically shared with other accounts so that specified
 portfolio.shareWithAccount('012345678901');
 ```
 
+### TagOptions
+
+TagOptions allow you to specify a key with a selection of values.  It enables end users to select an option for
+a certain key, for example an `ec2` instance type size. 
+
+```ts fixture=basic-portfolio
+const tagOptions = {
+  key1: [
+    { value: 'value1' },
```

Comment:
So, right now, a customer has to write `key: [{ value: 'value1' }, { value: 'value2' }]` instead of the much simpler `key: ['value1', 'value2']`, because of the off-chance that a value will be disabled. I would guess disabling values is not a very common operation...?

How about we made `TagOptions` a class with a method to disable values?

```ts
const tagOptions = new servicecatalog.TagOptions({
  key1: ['value1', 'value2'],
});
tagOptions.disable('key1', 'value2');
```

Thoughts on this?

Reply:
...
yes that is way better. We used guidance before that was probably L1 centric.  Although this would require more engineering on our part because we would need to now track the key-values states in the background.  I might try to pivot this out of the current PR since it will take more time.

Reply:
punting this to a new PR